models/platform-limits/src/limit-with-current/presets/with-warning-exceeded.ts

```diff
@@ -0,0 +1,5 @@
+import LimitWithCurrent from '../builder';
+
+const withWarningExceeded = () => LimitWithCurrent().limit(10).current(8);
```

Comment:
Shouldn't `current` be a greater number than `limit`?

Reply:
Thanks for the explanation.
I understand the rule you mention, but readers coming to this file will not know about it.

For maintainability purposes, I think it would be easier to understand if we use a larger value. As an alternative, we can also leave the `8` but add a comment about the 80% rule.

Reply:
Added comment https://github.com/commercetools/test-data/pull/655/commits/fcd86866594399f9f3a762d5eaa7b027d81aa686